src/sanitation.js

```diff
@@ -88,7 +88,7 @@ export const BLACKLISTED_TAGS = {
 
 /**
  * AMP elements allowed in AMP4EMAIL, modulo:
- * - amp-list, which cannot be nested.
+ * - amp-list, amp-state and amp-form, which cannot be nested.
```

Comment:
nit: amp-form before amp-list (alphabetical)

Reply:
Actually, `amp-form` should be allowed in the template output since it doesn't generate XHRs without user gesture. I'm not sure why it's not listed here, probably because the tag is actually `<form>` and not `<amp-form>`. I removed `amp-form` from the comment.